config.py.in

```diff
@@ -33,10 +33,13 @@ __with_spice__ = "@withspice@"
 __with_spice_web_client__ = "@withspicewebclient@"
 __version__ = "@kimchiversion@"
 __release__ = "@kimchirelease@"
+__runstatedir__ = "@runstatedir@"
 
 # Storage pool constant for read-only pool types
 READONLY_POOL_TYPE = ['iscsi', 'scsi', 'mpath']
 
+def get_libvirt_path():
+    return os.path.join(__runstatedir__, 'libvirt')
```

Comment:
Instead of setting `__runstatedir__` you can use `return os.path.join("@runstatedir@", 'libvirt')` directly as it was done in other parts of the code.